go.mod

```diff
@@ -36,3 +34,5 @@ require (
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
 	sigs.k8s.io/kube-storage-version-migrator v0.0.3
 )
+
+replace github.com/openshift/library-go v0.0.0-20210318152630-323ad8a8f7d8 => /Users/emoss/library-go
```

Comment:
maybe go mod silently ignored the `replace` directive, I don't see `audit.go` in the library-go changes. the `replace` maybe malformed.
I think it should be 
```
replace github.com/openshift/library-go => /Users/emoss/library-go
```

Reply:
The replace is ok I think. It is not passing make build.

Reply:
@tkashem and @sttts after correcting the build errors I see the audit package now

`go: found github.com/openshift/library-go/pkg/operator/apiserver/audit in github.com/openshift/library-go v0.0.0-20210318152630-323ad8a8f7d8`

Reply:
How can the replace be ok? How is the build system supposed to know what's in `/Users/emoss/library-go`?

Reply:
Please look at `go.mod` in https://github.com/openshift/apiserver-library-go/pull/46 to see how it's done. You need to push your changes somewhere so that they are publicly accessible.

Reply:
@stlaz this is only the proof PR. 
For the real PR the changes will come from library-go master. As this was a local change only to library-go which is not pushed, that is why the option was to use local dir.